scripts/build-catalogue.in

```diff
@@ -108,8 +108,20 @@ with TemporaryDirectory() as tmp:
         fd.write(cmake)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/BuildModules.cmake', tmp)
     shutil.copy2(f'@ARB_INSTALL_DATADIR@/generate_catalogue', tmp)
-    sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
-    sp.run('make',     shell=True, check=True, capture_output=not verbose)
-    shutil.copy2(f'{name}-catalogue.so', pwd)
+    try:
+        sp.run('cmake ..', shell=True, check=True, capture_output=not verbose)
+        sp.run('make',     shell=True, check=True, capture_output=not verbose)
+        shutil.copy2(f'{name}-catalogue.so', pwd)
+    except sp.CalledProcessError as e:
+        import sys, traceback as tb
+
+        sys.stdout.write("Build log:\n")
+        sys.stdout.write(e.stdout.decode())
```

Comment:
Should this be piped to stdout or stderr?

Reply:
Well it's not part of the error message, it's relaying the underlying `stdout` for completeness, so I'd say to `stdout`. If we're being run as shell script it'll both end up on the screen, and if we're being piped elsewhere the regular output and error output are available separately.

To sum up the pipe behavior:
* No error: Check verbosity whether to blurt out underlying log to `stdout`
* Error: Always blurt out underlying log to `stdout` ignoring verbosity and send stack trace and underlying error to `stderr`

Reply:
@brenthuisman what to do?

Reply:
stdout to stdout and errors to stderr sounds entirely reasonable!